package.json

```diff
@@ -47,15 +44,16 @@
         "react-dom": "^17.0.2",
         "react-helmet": "^6.1.0",
         "react-i18next": "^11.15.3",
-        "react-is": "^17.0.2",
         "react-paginate": "^8.1.0",
         "react-perfect-scrollbar": "^1.5.8",
         "react-ticker": "^1.3.1",
         "use-query-params": "^1.2.3",
-        "web-push-notifications": "^3.24.0",
-        "webpack": "^5.65.0"
```

Comment:
we need to check that removing this does not affect anything!

Reply:
I checked around as far as I could, everything was fine, how about we add it in devDeps instead? 
definitely we don't need webpack in deps.

Reply:
we can pass it to qa to test it completely
if they dont find any issues then we can proceed with it

Reply:
cool, will request their help on this